### PR TITLE
Replace grunt-postcss with postcss-cli

### PR DESCRIPTION
**Pull request by cvrebert**
Refs #19990.
CC: @XhmikosR @hnrch02

### CODE REVIEW

**Comment by amitjain24524** [comment]
k
On 20-Jun-2016 10:22 am, "Chris Rebert" <notifications@github.com> wrote:
>
> Refs #19990.
> CC: @XhmikosR @hnrch02
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/twbs/bootstrap/pull/20140
>
> Commit Summary
> Remove mq4-hover-shim for now
> Update npm shrinkwrap
> Replace grunt-postcss with postcss-cli
> Update npm shrinkwrap
> File Changes
> M Gruntfile.js (46)
> M docs/getting-started/build-tools.md (2)
> D grunt/autoprefixer-settings.js (31)
> M grunt/npm-shrinkwrap.json (1179)
> A grunt/postcss.js (42)
> M package.json (9)
> M scss/mixins/_hover.scss (19)
> Patch Links:
> https://github.com/twbs/bootstrap/pull/20140.patch
> https://github.com/twbs/bootstrap/pull/20140.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

File: scss/mixins/_hover.scss [hunk 1]
```diff
@@ -1,14 +1,15 @@
 @mixin hover {
-  @if $enable-hover-media-query {
-    // See Media Queries Level 4: http://drafts.csswg.org/mediaqueries/#hover
-    // Currently shimmed by https://github.com/twbs/mq4-hover-shim
-    @media (hover: hover) {
-      &:hover { @content }
-    }
-  }
-  @else {
+  // TODO: re-enable along with mq4-hover-shim
```
Comment (hnrch02) [inline scss/mixins/_hover.scss]: Would you add this back in a separate PR or was this an oversight?

**Comment by hnrch02** [comment]
Otherwise LGTM.

File: scss/mixins/_hover.scss (hunk 1 above)
Comment (cvrebert) [inline scss/mixins/_hover.scss]: Separate PR. Would need to first change mq4-hover-shim to implement PostCSS's standard plugin API.
Also depends on the answer to https://github.com/twbs/bootstrap/issues/19968#issuecomment-227212987

Comment (Puigcerber) [inline scss/mixins/_hover.scss]: What's the issue tracking this so I can subscribe? Cheers.

**Pull request merged**